content/events-api/migrate-to-event-platform.md

```diff
@@ -0,0 +1,98 @@
+
+As with the subscriber, you can use Postman or another tool like Curl. The URL to call is `https://event.prd.sdk.akeneo.cloud/api/v1/subscribers/{subscriber_id}/subscriptions`. The headers remain the same as before.
+
+For more details, refer to the documentation for the endpoint [here](https://storage.googleapis.com/akecld-prd-sdk-aep-prd-api-assets/openapi_specification.html#tag/Subscription/operation/create_subscription). To summarize, here’s an example of the payload you need to send:
+
+```json
+
+{
+    "source": "pim",
+    "subject": "https://pim.url",
+    "events": [
+        "com.akeneo.pim.v1.product.created",
+        "com.akeneo.pim.v1.product.updated.delta",
+        "com.akeneo.pim.v1.product.deleted",
+        "com.akeneo.pim.v1.product-model.created",
+        "com.akeneo.pim.v1.product-model.updated.delta",
+        "com.akeneo.pim.v1.product-model.deleted"
+    ],
+    "type": "https",
+    "config": {
+        "url": "https://destination.url",
+        "secret": {
+            "primary": "averysecretprimarysecret",
+            "secondary": "can be empty, only used to ease secret rotation"
+        }
+    }
+}
+```
+
+This example includes all the events necessary to maintain a similar system to what you had before. You can choose fewer or additional events as needed. A complete list of available events can be found [here](https://api.akeneo.com/event-platform/available-events.html#available-events).
+
+Now, all relevant events should be sent. To test the setup, simply create, modify, or delete a product and check your destination to see if you receive the corresponding event.
+
+
+## Development adjustments
+### Product UUID and removal of identifier
+In the previous Event API, you had the option to send product information using either UUID or identifier. With the Event Platform, only the UUID is transmitted. If you were using product identifiers, you will need to transition to using UUIDs.
+
+This change will not affect product model events.
+
+### Differential payload (delta) instead of full payload
+In the old Event API, each event related to an update provided a complete snapshot of the object (simple product, model, or variant), including a wealth of data. With the Event Platform and the `*.updated.delta` events, you will now receive only the data that has changed. The format for these new events is detailed in the documentation [here](https://api.akeneo.com/event-platform/available-events.html).
+
+If you need to access all product or product model information upon receiving events, you will need to utilize the PIM REST API.
+
+### Filtering events triggered by yourself
+With the Event Platform, you will receive events triggered by creation, modification, or deletion actions, even if your Connection initiated those actions. Therefore, it is essential to implement a mechanism to filter out these self-triggered events to avoid entering an infinite loop.
```

Comment:
``` suggestion
With the Event Platform, you will receive events triggered by creation, modification, or deletion actions, even if your Connection initiated those actions. Therefore, it is essential to implement a mechanism to filter out these self-triggered events to avoid an infinite loop.
```